Source/AliveLibAE/Abe.cpp

```diff
@@ -9125,10 +9125,13 @@ short Abe::DoGameSpeak_45AB70(int input)
     short nextState = -1;
     if ((sInputObject_5BD4E0.field_0_pads[sCurrentControllerIndex_5C1BBE].field_0_pressed & eChant) == eChant)
     {
-        field_124_gnFrame = sGnFrame_5C1B84 + 90;
-        SND_SEQ_PlaySeq_4CA960(SeqId::MudokonChant1_10, 0, 1);
-        field_120_state = 0;
-        nextState = eAbeStates::State_112_Chant_45B1C0;
+        if (!(field_168_ring_pulse_timer && field_16C_bHaveShrykull))
```

Comment:
I think it might be worth adding a comment on this new if statement saying something like:

// Fix OG bug where abe doesn't transform into shrykull when you use gamespeak and then immediately chant 

Since this will be the first game play changing bug we've fixed that is OG?

Reply:
can do but i think we already had some OG fixes/logic changes (excluding crash bug fixes)